lib/monitor/rtime_windows.go

```diff
@@ -0,0 +1,21 @@
+// +build windows
+
+package monitor
+
+import (
+	"github.com/dedis/cothority/lib/dbg"
+	"syscall"
+)
+
+// Returns the system and the user CPU time used by the current process so far.
+func getRTime() (tSys, tUsr float64) {
+	var creationTime, exitTime, kernelTime, userTime syscall.Filetime
```

Comment:
Nicely found: it does not show in the [syscall package](https://golang.org/pkg/syscall/).